samples/csharp_dotnetcore/05.multi-turn-prompt/Bots/DialogBot.cs

```diff
@@ -10,41 +10,41 @@
 
 namespace Microsoft.BotBuilderSamples
 {
-    // This IBot implementation can run any type of Dialog. The use of type parameterization is to allows multiple different bots
+    // This IBot implementation can run any type of Dialog. The use of type parametrization is to allows multiple different bots
```

Comment:
I think parameterization is the correct spelling.  you change introduces a spelling error, no?

Reply:
sorry, parameterization wasn't in the dictionary I was using to check spelling on the source code, added the word and reverted the change.